src/AppForeach.Framework.MassTransit/ConsumerConfigurationBuilder.cs

```diff
@@ -0,0 +1,23 @@
+﻿using MassTransit;
+
+namespace AppForeach.Framework.MassTransit;
+
+public class ConsumerConfigurationBuilder<TConsumer> : IConsumerConfigurationBuilder<TConsumer>
+    where TConsumer : class
+{
+    private readonly List<Action<IConsumerConfigurator<TConsumer>>> actions = [];
+
+    public IConsumerConfigurationBuilder<TConsumer> Configure(Action<IConsumerConfigurator<TConsumer>> action)
+    {
+        actions.Add(action);
```

Comment:
Check for `null` unless you have #nullable enabled

Reply:
fixed